packages/@webex/webex-core/src/lib/services-v2/interceptors/service.js

```
@@ -0,0 +1,101 @@
+  /**
+   * Normalizes request options relative to service identification.
+   *
+   * @param {Object} options - The request PTO.
+   * @returns {Object} - The mutated request PTO.
+   */
+  normalizeOptions(options) {
+    // Validate if the api property is used.
+    if (options.api) {
+      // Assign the service property the value of the api property if necessary.
+      options.service = options.service || options.api;
+      delete options.api;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)